source/installation/file-location.txt

```diff
@@ -0,0 +1,74 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Review the location of files needed when working with Relational Migrator. 
```

Comment:
This intro doesn't adequately explain why users might care about this. In most cases people won't need this info, but it can be useful in a few situations:
 - Deploying additional JDBC drivers
 - Modifying application configuration files
 - Reviewing log files